packages/core/src/fonts.css

```diff
@@ -92,10 +81,8 @@
 	font-style: normal;
 	font-weight: 600;
 	src: local('Open Sans SemiBold'), local('OpenSans-SemiBold'),
-		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.semibold.latin.woff2)
-			format('woff2'),
-		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.semibold.latin.latin-ext.woff)
-			format('woff');
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.semibold.latin.woff2) format('woff2'),
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.semibold.latin.latin-ext.woff) format('woff');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Quote this URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aquoted-url)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this protocol-relative URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aprotocol-relative-url)